Sources/SwiftBuild/SWBBuildServiceSession.swift

```diff
@@ -582,6 +582,10 @@ public final class SWBBuildServiceSession: Sendable {
         try await service.send(request: DeveloperPathRequest(sessionHandle: uid)).value
     }
 
+    public func preferredRunDestination(forPlatform platformName: String) async throws -> RunDestinationInfo? {
```

Comment:
This should return an `SWBRunDestinationInfo`, which is the public API cover type for `RunDestinationInfo`.

Reply:
Also, why can it return nil? It doesn't seem like it can in practice.

Reply:
OK, I see. I wasn’t sure what the rationale for not using the protocol types in `SwiftBuild` was and tried to avoid unnecessary work but now using `SWBRunDestinationInfo`. And good catch with `nil`.